tests/functional/spec/supplementary_data_calculated_summary/supplementary_data_calculated_summary.spec.js

```diff
@@ -0,0 +1,259 @@
+import { assertSummaryItems, assertSummaryTitles, assertSummaryValues, listItemComplete, click, verifyUrlContains } from "../../helpers";
```

Comment:
I would follow the test survey name here, it does test `repeating blocks` a few times here as well.